python/pyspark/ml/classification.py

```diff
@@ -183,7 +191,7 @@ def getThresholds(self):
         If :py:attr:`thresholds` is set, return its value.
         Otherwise, if :py:attr:`threshold` is set, return the equivalent thresholds for binary
         classification: (1-threshold, threshold).
-        If neither are set, throw an error.
```

Comment:
Actually now that I'm back from traveling I think reverting the doc change is the right thing to do since that is what the Scala side does (I misread it earlier).